tokens/tokens-eth.json

```diff
@@ -15247,31 +15248,6 @@
 			"youtube": ""
 		}
 	},
-	{
```

Comment:
Can you merge the master branch to this pr so you don't remove this token?

Reply:
My apologies, but I am not sure what else to do to make that happen. I definitely don't want to remove the token. I will research how to do this, but any advice or assistance you can offer would be very much appreciated. Thank you.